.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve the stackstorm docker image
+
+---
+
+**Describe the problem**
+A clear and concise description of what the problem is.
+
+**Versions**
+ - Host OS: [e.g. Ubuntu 16.04]
+ - docker: [e.g. 18.03.1-ce]
+ - docker-compose: [e.g. 1.21.1]
+ - stackstorm/stackstorm image: [e.g. v2.7.1]
+
+**To Reproduce**
+If necessary, please specify steps to reproduce the behavior.
+
+**Screenshots**
```

Comment:
TBH I don't really think _any_ screenshots will help us in Docker.

Logs and code - absolutely yes.